Dockerfile

```diff
@@ -28,11 +28,33 @@ RUN apt-get -y install python build-essential libglib2.0-dev libpixman-1-dev
 RUN apt-get -y install libfdt-dev zlib1g-dev
 # Not required or specified anywhere but supress build warnings
 RUN apt-get -y install flex bison
-RUN "qemu-${QEMU_VERSION}/configure" --static --target-list=arm-softmmu
+RUN "qemu-${QEMU_VERSION}/configure" --static --target-list=arm-softmmu,aarch64-softmmu
 RUN make -j$(nproc)
 
 RUN # Strip the binary, this gives a substantial size reduction!
-RUN strip "arm-softmmu/qemu-system-arm"
+RUN strip "arm-softmmu/qemu-system-arm" "aarch64-softmmu/qemu-system-aarch64"
+
+
+# Build stage for fatcat
+FROM debian:stable-slim AS fatcat-builder
+ARG FATCAT_VERSION=1.1.0
+ENV FATCAT_TARBALL="v${FATCAT_VERSION}.tar.gz"
+WORKDIR /fatcat
+
+RUN # Update package lists
+RUN apt-get update
+
+RUN # Pull source
+RUN apt-get -y install wget
+RUN wget "https://github.com/Gregwar/fatcat/archive/${FATCAT_TARBALL}"
```

Comment:
We're just trusting GitHub to serve us the right file here. Currently all external resources have some sort of validation to guarantee we're getting what we expect. 

The QEMU source is verified with GPG and the versatilepb kernel and Raspbian image are verified against checksums.

It would be great if we verified the fatcat tarball against a checksum too.

Reply:
I agree, we should definitely do that. I'll make that change tomorrow.